<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DVA-F0L9N9R-L" name="Default test" tests="2" failures="0" timestamp="20 Jun 2017 19:54:42 GMT" time="75.602" errors="0">
  <testcase name="@BeforeClass buildUp" time="0.0" classname="com.vh.ui.tests.CurrentLabsTest">
    <failure type="java.lang.AssertionError" message="Failed to identify UserName text field expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Failed to identify UserName text field expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.vh.ui.actions.ApplicationFunctions.capellaLogin(ApplicationFunctions.java:84)
	at com.vh.ui.tests.CurrentLabsTest.buildUp(CurrentLabsTest.java:48)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:514)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:215)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:178)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- @BeforeClass buildUp -->
  <testcase name="@AfterClass tearDown" time="0.0" classname="com.vh.ui.tests.CurrentLabsTest">
    <skipped/>
  </testcase> <!-- @AfterClass tearDown -->
  <testcase name="verify_CurrentLabsPage" time="0.003" classname="com.vh.ui.tests.CurrentLabsTest">
    <skipped/>
  </testcase> <!-- verify_CurrentLabsPage -->
  <testcase name="verify_AddAddLabResultsPopup" time="0.004" classname="com.vh.ui.tests.CurrentLabsTest">
    <skipped/>
  </testcase> <!-- verify_AddAddLabResultsPopup -->
</testsuite> <!-- Default test -->
